Tools/ardupilotwaf/boards.py

```diff
@@ -488,6 +488,10 @@ def configure_env(self, cfg, env):
             'uavcan',
         ]
 
+        env.AP_LIBRARIES += [
+            'AP_UAVCAN',
```

Comment:
Ok, I see you have struggling with this, sorry about that. The correct place for this is in line 168, you just need to put `'AP_UAVCAN',` there before the other one.